tests/foreman/virtwho/ui/test_nutanix.py

```diff
@@ -237,3 +240,65 @@ def test_positive_prism_central_prism_flavor_option(self, default_org, session,
             assert get_configure_option('prism_central', config_file) == 'true'
             session.virtwho_configure.delete(name)
             assert not session.virtwho_configure.search(name)
+
+    @pytest.mark.tier2
+    def test_positive_ahv_internal_debug_option(self, default_org, session, form_data):
+        """Verify ahv_internal_debug option by hammer virt-who-config"
+
+        :id: af92b814-fe7a-4969-b2ab-7b68859de3ce155
+
+        :expectedresults:
+            enable-ahv-debug option has been set to false
+            ahv_internal_debug bas been set to false in virt-who-config-X.conf
+            warning message exist in log file /var/log/rhsm/rhsm.log
+            ahv_internal_debug option can be updated
+            message Host UUID {system_uuid} found for VM: {guest_uuid} exist in rhsm.log
+            ahv_internal_debug bas been set to true in virt-who-config-X.conf
+            warning message does not exist in log file /var/log/rhsm/rhsm.log
```

Comment:
same numbering the steps can be done as above.

Reply:
> same numbering the steps can be done as above.

@shubhamsg199 Have updated.